package.json

```diff
@@ -13,125 +13,80 @@
   },
-  "devDependencies": {
+  "private": true,
+  "dependencies": {
     "@a-ui/acpaas": "^4.1.0",
     "@a-ui/core": "^4.1.0",
-    "@a-ui/flexboxgrid": "~1.0.1",
     "@acpaas-ui/js-date-utils": "^1.0.6",
-    "@acpaas-ui/js-notification-store": "1.0.1-alpha.5",
-    "@acpaas-ui/schematics": "0.0.8",
-    "@angular-devkit/build-angular": "~0.10.7",
-    "@angular-devkit/build-ng-packagr": "~0.8.0",
-    "@angular-redux/store": "~9.0.0",
-    "@angular/animations": "^6.1.0",
-    "@angular/cli": "~6.1.0",
-    "@angular/common": "^6.1.0",
-    "@angular/compiler": "^6.1.0",
-    "@angular/compiler-cli": "^6.1.0",
-    "@angular/core": "^6.1.0",
-    "@angular/forms": "^6.1.0",
-    "@angular/http": "^6.1.0",
-    "@angular/language-service": "^6.1.0",
-    "@angular/platform-browser": "^6.1.0",
-    "@angular/platform-browser-dynamic": "^6.1.0",
-    "@angular/router": "^6.1.0",
-    "@commitlint/cli": "~7.0.0",
-    "@commitlint/config-angular": "~7.0.1",
-    "@commitlint/config-conventional": "~7.0.1",
-    "@schematics/angular": "^0.6.5",
-    "@types/jasmine": "~2.8.6",
+    "@acpaas-ui/js-notification-store": "^1.0.4",
+    "@angular-redux/store": "^10.0.0",
+    "@angular/animations": "~8.2.14",
+    "@angular/common": "~8.2.14",
+    "@angular/compiler": "~8.2.14",
+    "@angular/core": "~8.2.14",
+    "@angular/forms": "~8.2.14",
+    "@angular/platform-browser": "~8.2.14",
+    "@angular/platform-browser-dynamic": "~8.2.14",
+    "@angular/router": "~8.2.14",
+    "@ngrx/effects": "^8.6.0",
+    "@ngrx/store": "^8.6.0",
+    "@ngrx/store-devtools": "^8.6.0",
+    "angular2-highlight-js": "^5.0.2",
+    "cookieconsent": "^3.1.1",
+    "esri-leaflet": "^2.3.3",
+    "hammerjs": "^2.0.8",
+    "highlight.js": "^9.18.1",
+    "inputmask": "^5.0.3",
+    "leaflet": "^1.6.0",
+    "leaflet-draw": "^1.0.4",
+    "lodash-es": "^4.17.15",
+    "marked": "^0.8.0",
+    "ng2-ckeditor": "^1.2.7",
+    "redux": "4.0.1",
+    "rimraf": "^3.0.2",
+    "rxjs": "~6.4.0",
+    "short-unique-id": "^1.1.1",
+    "tslib": "^1.10.0",
+    "zone.js": "~0.9.1"
+  },
+  "devDependencies": {
+    "@angular-devkit/build-angular": "~0.803.25",
+    "@angular-devkit/build-ng-packagr": "~0.803.25",
+    "@angular/cli": "~8.3.20",
+    "@angular/compiler-cli": "~8.2.14",
+    "@angular/language-service": "~8.2.14",
+    "@types/jasmine": "~3.3.8",
     "@types/jasminewd2": "~2.0.3",
-    "@types/lodash-es": "~4.17.0",
-    "@types/marked": "0.4.0",
+    "@types/marked": "^0.7.3",
     "@types/node": "~8.9.4",
-    "@types/sinon": "^7.5.1",
-    "@userfrosting/merge-package-dependencies": "~1.2.0-rc.2",
-    "angular2-highlight-js": "~5.0.2",
-    "codelyzer": "~4.3.0",
-    "colors": "1.3.0",
-    "command-line-args": "~5.0.2",
-    "cookieconsent": "~3.1.1",
-    "core-js": "~2.5.4",
-    "cpx": "1.5.0",
-    "esri-leaflet": "^2.1.4",
-    "hammerjs": "~2.0.8",
-    "inputmask": "~3.3.11",
-    "jasmine-core": "~2.99.1",
+    "@types/sinon": "^7.5.2",
```

Comment:
Ik zou aanraden om packages waar we zelf geen invloed op hebben altijd vast te zetten met maximaal een `~`. Je kan nooit voorspellen hoe goed die packages semver respecteren en voor je het weet zit je toch met een breaking change in een minor waardoor je eerst een half uur het probleem moet zoeken voordat je kan starten met het ontwikkelen van een package. 

Je kan in de `.npmrc` file instellen dat dit standaard met een ~ geïnstalleerd moet worden.